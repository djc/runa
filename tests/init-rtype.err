init-rtype.rns [2.27]: method '__init__' must return type 'void'
    def __init__(self) -> i32:
                          ^
