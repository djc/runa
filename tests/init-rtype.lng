class test:
	def __init__(self: test) -> int:
		return 1
