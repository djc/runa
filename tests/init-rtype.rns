class test:
	def __init__(self) -> i32:
		return 1
