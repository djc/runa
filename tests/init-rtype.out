tests/init-rtype.lng [2.30]: __init__() method return type must be 'void'
    def __init__(self: test) -> int:
                                ^
